usr/sbin/truenas-install

```diff
@@ -1012,6 +1032,8 @@ main()
     [ -f /etc/hostid ] || zgenhostid
     modprobe zfs
 
+    [ -f /etc/version ] && report_installation
```

Comment:
Can we do all this in the background and suppress all output? log ot a file?

Reply:
Yea, I'd agree here. No reason to block the installer from being able to launch